TravisBotTests/TravisBotTests.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  TravisBotTests.swift
+//  TravisBotTests
+//
+//  Created by Douglas Garcia Barroso on 22/03/20.
+//  Copyright © 2020 Douglas Garcia Barroso. All rights reserved.
+//
+
+import XCTest
+@testable import TravisBot
+
+class TravisBotTests: XCTestCase {
+
+    override func setUp() {
+    }
+
+    override func tearDown() {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/dbgarcia/danger-bot/blob/4ad3f65193926f1211a9cfb888b46cbbba684b48/TravisBotTests/TravisBotTests.swift#L17"></span>Method ‘tearDown()’ should call to super function<br />
<code>overridden_super_call</code> <code>TravisBotTests.swift:17</code></del></td>
    </tr>
  </tbody>
</table>